pyHS100/cli.py

```diff
@@ -41,11 +44,17 @@ def cli(ctx, ip, host, debug, bulb, plug):
     if ip is not None and host is None:
         host = ip
 
+    if alias is not None and host is None:
+        click.echo("Alias is given, using discovery to find host %s" %
+                   alias)
+        host = find_host_from_alias(alias=alias)
+        print("Found hostname is {}".format(host))
+
+
```

Comment:
We do not have hound-ci active at the moment (which i hope we'll get enabled soonish) to notify about code-style issues, but here is an extra newline.